app/routes/company/components/CompanyDetail.tsx

```diff
@@ -41,23 +49,25 @@ const queryString = (companyId?: EntityId) =>
 const CompanyDetail = () => {
   const [viewOldEvents, setViewOldEvents] = useState(false);
 
-  const { companyId } = useParams<{ companyId: string }>();
+  const { companyId } = useParams<{ companyId: string }>() as {
+    companyId: string;
+  };
   const showFetchMoreEvents = useAppSelector((state) =>
     selectPagination('events', {
       queryString: queryString(companyId),
     })(state),
```

Comment:
Oops, I forgot this! This should be updated to `paginationNext` to work when events slice in updated to `legoAdapter`. I'm not sure it even works now with the changes I made to the companies slice.